### DISCUSSION TITLE
Wich feature can i use to classify timeseries?

### DISCUSSION THREAD

**Discussion by muriloasouza** [fraunhoferportugal/tsfel#141] [Q&A]
I have a dataset with `N` timeseries and `24` hourly measurements of electricity energy consumption (a single day). So, it is a dataset with  `N` rows and `24` columns. Consider these samples as class `0`.

I need to generate more `N` rows, multiplying each of those rows of my original dataset by a random reduction factor between 0.1 and 0.9. These new samples are class `1`.

The problem is to solve this timeseries classification problem. One sample from class `0` that has a low energy consumption pattern, could be missclassified as class `1` (since class `1` timeseries has the same format/shape as class `0`, but with the reduction factor applied). Wich feature could i use to improve my model predicton?

I have used the mean value wich improved a bit, but still far from acceptable. Anything else i could try?

**Comment by dmfolgado** [comment]
As a general guideline, the features depend on the problem at hand; therefore, possessing domain knowledge about the problem and its associated data is particularly important.

Based on your description, I recommend exploring features from the statistical set, especially as you are generating new samples for class 1 by scaling the data
